extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -108,58 +124,12 @@ class AmpInstagram extends AMP.BaseElement {
         setStyles(iframe, {
           'opacity': 1,
         });
-
-        // Hide the initial rendered image to avoid overlaying videos.
-        setStyles(this.placeholderWrapper_, {
-          'display': 'none',
-        });
+        this.togglePlaceholder(false);
```

Comment:
Dropped from here and reset `ElementProto.layoutCount_` to 0 if relayout is needed inside `ElementProto.unlayoutCallback`